Dockerfile

```diff
@@ -18,14 +26,15 @@ COPY docker/nginx.conf /etc/nginx/conf.d/django.conf
 COPY docker/supervisor.conf /etc/supervisor/conf.d/molo.conf
 COPY docker/supervisord.conf /etc/supervisord.conf
 
+
 EXPOSE 8000
 
 CMD ["docker-start.sh"]
 
 COPY . /app
-RUN pip install -e . && \
-    pip install -r pip-freeze.txt && \
-    pip install -r pip-freeze-praekelt.txt && \
-    npm install -g less coffee-script && \
-    django-admin collectstatic --noinput &&\
-    USE_DEFAULT_CACHE=True django-admin compress
+RUN pip install -e . &&\
+pip install -r pip-freeze.txt &&\
+pip install -r pip-freeze-praekelt.txt &&\
+npm install -g less coffee-script &&\
+django-admin collectstatic --noinput &&\  
```

Comment:
I think just tab it to fix the alignment and remove the empty spaces at the end of this line.